source/js/components/DatePicker/DatePicker.tsx

```diff
@@ -27,47 +27,51 @@ export const DatePicker = ({ availableDates, date, required, onDateSelected }: D
     <>
       <div className="o-grid-6@md">
         <div className="form-group">
-          <label htmlFor="selectDate">
-            Datum
-            <br />
-          </label>
-          <select
-            id="selectDate"
-            value={selectedDate}
-            onChange={handleDateChange}
-            className="c-select"
-            required={required}>
-            <option value={''} key={'default'}>
-              Välj datum
-            </option>
-            {dates.map((dateValue) => (
-              <option key={dateValue}>{dateValue}</option>
-            ))}
-          </select>
-          <br />
+          <div className="c-select c-select--md c-field">
+            <label htmlFor="selectDate">Datum</label>
+            <div className="u-position--relative">
+              <select id="selectDate" value={selectedDate} onChange={handleDateChange} required={required}>
+                <option value={''} key={'default'} className="c-select__option">
+                  Välj datum
+                </option>
+                {dates.map((dateValue) => (
+                  <option key={dateValue} className="c-select__option">
+                    {dateValue}
+                  </option>
+                ))}
+              </select>
+              <i className="c-icon c-select__icon c-icon--size-md material-icons" translate="no" role="img">
+                expand_more
+              </i>
+            </div>
+          </div>
```

Comment:
I think there is a lot of duplicate code here. For example, 

```js
<div className="o-grid-6@md">
        <div className="form-group">
          <div className="c-select c-select--md c-field">
            <label htmlFor="selectDate">Datum</label>
            <div className="u-position--relative">
              <select id="selectDate" value={selectedDate} onChange={handleDateChange} required={required}>
                <option value={''} key={'default'} className="c-select__option">
                  Välj datum
                </option>
                {dates.map((dateValue) => (
                  <option key={dateValue} className="c-select__option">
                    {dateValue}
                  </option>
                ))}
              </select>
              <i className="c-icon c-select__icon c-icon--size-md material-icons" translate="no" role="img">
                expand_more
              </i>
            </div>
          </div>
        </div>
      </div>
```

This could be refactored to one component and used twice like:

```js

// DatePicker component

const DatePicker = () => {

const handleDateChane = () => {...}

const handleTimeChange = () => {...}

  return (
    <Picker label="a label" id="selectDate" onChange={handleDateChange} values={dates array}/>
    <Picker label="a label" id="selectTime" onChange={handleTimeChange} values={times array}/>
  );
}

```

Reply:
Created a simple component for the select.